zproject/default_settings.py

```diff
@@ -238,6 +238,7 @@
 # ERROR_BOT sends Django exceptions to an "errors" stream in the
 # system realm.
 ERROR_BOT: Optional[str] = None
+ERROR_BOT_SUBDOMAIN: Optional[str] = None
```

Comment:
I don't think we need a new setting for this; we should just use `STAFF_SUBDOMAIN` or `SYSTEM_BOT_REALM`, probably the former.